joomla/joomla-cms
# Check for MCrypt in SysInfo

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/9507.
#### Summary of Changes

Adds php mcrypt extension information to sysinfo.

![image](https://cloud.githubusercontent.com/assets/9630530/13970969/54a4d5e8-f084-11e5-8a04-25e4da7b6894.png)
#### Testing Instructions
1. Use Joomla 3.5.0
2. Aply patch
3. Go to sysinfo and check if in php settings separator there is a new line "MCrypt Enabled"
#### Additional information

PR for adding this info to the installer: https://github.com/joomla/joomla-cms/pull/9508
